06/PointerVectorExample.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+#include <vector>
+#include "PointerVectorExample.h"
+#include "Score.h"
+
+using namespace std;
+
+namespace samples
+{
+	void PointerVectorExample()
+	{
+		vector<Score*> myScoreList;
+		myScoreList.reserve(5);
+
+		myScoreList.push_back(&Score(30, "class 1"));
```

Comment:
Score object is temporary so doing it... is dangerous.. you will have to create

Score score1; to scoreXx.. then use &  but that's prolly usually not right way of doing anything

let's do new Score() and ... (see below)